--- CMakeModules/FindImageMagickLib.cmake	2009-06-10 04:30:37.000000000 +0800
+++ CMakeModules/FindImageMagickLib.cmake	2009-06-30 20:40:08.000000000 +0800
@@ -83,6 +83,7 @@
   /usr/local/lib
   /usr/openwin/lib
   /usr/lib
+  /usr/lib64
   )
 #MESSAGE("DBG IMAGEMAGICKLIB_X11_LIBRARY=${IMAGEMAGICKLIB_X11_LIBRARY}")
 ENDIF(NOT APPLE)

--- src/CMakeLists.txt	2009-06-10 04:29:21.000000000 +0800
+++ src/CMakeLists.txt	2009-06-30 20:36:50.000000000 +0800
@@ -12,7 +12,7 @@
 -lpthread
 -lz
 -lutil
-libtuxcap.so.4.0)
+-ltuxcap)
 
 SET_TARGET_PROPERTIES(${CurrentExe}
                              PROPERTIES
